<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editinformation</class>
 <widget class="QMainWindow" name="editinformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
image: url(:/pishvazpic.JPG);
background-color: rgb(192, 221, 190);
font: 40px
}
QPushButton#pushButton_back{
background-color: black;
color: white;
font: bold 15px;
border-radius: 10px;
transition: 0.2s;
}
QPushButton#pushButton_back:hover {
background-color: rgb(80,80,80);
font: bold 17px;
}
QPushButton#pushButton_back:pressed {
background-color: rgb(50,50,50);
}

QPushButton#pushButton_back:focus{
box-shadow: 0 0 5px rgba(0,0,255,0.5);
font: bold 15px;
Width: 150px;
}
QPushButton#pushButton_select_photo{
background-color: black;
color: white;
font: bold 15px;
border-radius: 10px;
transition: 0.2s;
}
QPushButton#pushButton_select_photo:hover {
background-color: rgb(80,80,80);
font: bold 17px;
}
QPushButton#pushButton_select_photo:pressed {
background-color: rgb(50,50,50);
}

QPushButton#pushButton_select_photo:focus{
box-shadow: 0 0 5px rgba(0,0,255,0.5);
font: bold 15px;
Width: 150px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_username">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 66, 51);</string>
    </property>
    <property name="text">
     <string>Username :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_username">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 15px;
border-radius: 8px;
border: 1px solid #0640ff;
padding: 5px;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="placeholderText">
     <string>Enter username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fname">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 15px;
border-radius: 8px;
border: 1px solid #0640ff;
padding: 5px;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="placeholderText">
     <string>Enter first name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_password">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 66, 51);</string>
    </property>
    <property name="text">
     <string>Password :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 15px;
border-radius: 8px;
border: 1px solid #0640ff;
padding: 5px;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="placeholderText">
     <string>Enter password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_lname_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 66, 51);</string>
    </property>
    <property name="text">
     <string>Firstname :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_lname">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 66, 51);</string>
    </property>
    <property name="text">
     <string>Lastname :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_lname">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 15px;
border-radius: 8px;
border: 1px solid #0640ff;
padding: 5px;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="placeholderText">
     <string>Enter last name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 66, 51);</string>
    </property>
    <property name="text">
     <string>Email :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_email">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 15px;
border-radius: 8px;
border: 1px solid #0640ff;
padding: 5px;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="placeholderText">
     <string>Enter your email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 66, 51);</string>
    </property>
    <property name="text">
     <string>Skills :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_skills">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 15px;
border-radius: 8px;
border: 1px solid #0640ff;
padding: 5px;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="placeholderText">
     <string>Enter your skills</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 66, 51);</string>
    </property>
    <property name="text">
     <string>nationality :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_na">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 15px;
border-radius: 8px;
border: 1px solid #0640ff;
padding: 5px;
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="placeholderText">
     <string>Enter your nationality</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>441</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>5</pointsize>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="toolTipDuration">
     <number>16</number>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="locale">
     <locale language="Duala" country="Cameroon"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="lineEdit_school">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>45</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string notr="true"/>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
</string>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string notr="true">Enter your school or college name</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_start_year">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2010</number>
     </property>
     <property name="maximum">
      <number>2024</number>
     </property>
     <property name="value">
      <number>2019</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_end_year">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>140</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2020</number>
     </property>
     <property name="maximum">
      <number>2030</number>
     </property>
     <property name="value">
      <number>2030</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>15</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(54, 66, 51);</string>
     </property>
     <property name="text">
      <string>School or College / University :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(54, 66, 51);</string>
     </property>
     <property name="text">
      <string>Start year :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(54, 66, 51);</string>
     </property>
     <property name="text">
      <string>End year (or expected) :</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>270</y>
      <width>211</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextBrowserInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>270</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 66, 51);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Birth date :</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>300</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="date">
     <date>
      <year>2019</year>
      <month>3</month>
      <day>3</day>
     </date>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_select_photo">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>460</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>450</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
border-top-left-radius: 40px;
border-bottom-right-radius: 40px;</string>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>918</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
